dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/system/notification/NotificationMap.java

```diff
@@ -29,57 +29,82 @@
 
 import static java.util.Arrays.stream;
 import static java.util.Collections.unmodifiableMap;
+import static java.util.stream.Collectors.toMap;
 
-import java.util.Deque;
-import java.util.EnumMap;
-import java.util.LinkedList;
-import java.util.Map;
+import java.util.*;
```

Comment:
import collapse

Reply:
:+1: forgot to set this after my spring clean IDE reinstall - will do now